ci-operator/step-registry/openshift/microshift/e2e/reboot-tests/verify/openshift-microshift-e2e-reboot-tests-verify-commands.sh

```diff
@@ -27,39 +27,27 @@ Host ${INSTANCE_PREFIX}
   HostName ${IP_ADDRESS}
   IdentityFile ${CLUSTER_PROFILE_DIR}/ssh-privatekey
   StrictHostKeyChecking accept-new
-  ServerAliveInterval 30
-  ServerAliveCountMax 1200
 EOF
 chmod 0600 "${HOME}"/.ssh/config
 
-timeout=1200 # 20 minute wait.  
+# Steps may not be used more than once in a test, so this block duplicates the behavior of wait-for-ssh for reboot tests.
+timeout=300 # 5 minute wait.
+>&2 echo "Polling ssh connectivity before proceeding.  Timeout=$timeout second"
 start=$(date +"%s")
-until ssh "${INSTANCE_PREFIX}" 'true'; do
-  if (( $(date +"%s") - start >= timeout )); then
-    echo "timed out out waiting for ssh connection" >&2
+until ssh "${INSTANCE_PREFIX}" 'sudo systemctl start microshift';
```

Comment:
Do we need a loop here? If MicroShift cannot be started once, why would it be started on the second attempt?